units/systemd-initctl.socket

```diff
@@ -15,5 +15,5 @@ Before=sockets.target
 
 [Socket]
 ListenFIFO=/run/systemd/initctl/fifo
-Symlinks=/dev/initctl
+Symlinks=/dev/initctl /run/initctl
```

Comment:
the only reason we use the /run/systemd/initctl/fifo path for the fifo is that maintaining one in /dev is awful. However, if upstream sysvinit moved the socket, then we should just move ours too, i.e. make /run/initctl the main path we listen on, and then only do the /dev/initctl as symlink, as before

Reply:
should we slightly reword the Description in systemd-initctl.{service,socket}. It currently says
```
Description=/dev/initctl Compatibility Daemon
Description=/dev/initctl Compatibility Named Pipe
```
I would probably just drop the '/dev/' prefix

Reply:
> I would probably just drop the '/dev/' prefix

Done!